doc/ChangeLog

```diff
@@ -1,4 +1,98 @@
 ===============================================================
+Tag name:  ctsm1.0.dev027
+Originator(s):  negins (Negin Sobhani,UCAR/CSEG,303-497-1224)
+Date: Tue Feb 19 12:57:12 MST 2019
+One-line Summary: Non-constant time initialization for soil hydrology types are moved to SoilHydrologyType.F
+
+Purpose of changes
+------------------
+
+[Fill this in with details]
+Initalization of time-varying `zwt`, `zwt_perched`, and `frost_table` are moved to `InitCold` in `SoilHydrologyType`.
+
+Previously, `zwt`, `zwt_perched`, and `frost_table` were initialized in `SoilHydrologyInitTimeConstMod`.
+
+Bugs fixed or introduced
+------------------------
+Issues fixed (include CTSM Issue #575)
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+None
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm5_0
+
+[ ] clm4_5
+
+Notes of particular relevance for users
+---------------------------------------
+
+Caveats for users (e.g., need to interpolate initial conditions):
```

Comment:
Please explicitly answer the questions in this section; answering "none" is fine. (I know not all entries are consistent in this respect.)

Reply:
Thanks for this comment. I've added the "none" for all the questions in this section.